### PR TITLE
Teamstatsandorganization

### PR DESCRIPTION
**Pull request by WilliamLampke**

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -289,11 +290,76 @@ def worst_offense
     
     lowest_average = team_average.min_by{ |id, average| average }
```
Comment (DarbySmith) [inline lib/stat_tracker.rb:291]: This should be in worst_offense

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -289,11 +290,76 @@ def worst_offense
...
 
-    @teams_data.each do |row|
-      if lowest_average[0] == row[:team_id]
-        return row[:teamname]
```
Comment (DarbySmith) [inline lib/stat_tracker.rb:294]: This should be in worst_offense

Comment (DarbySmith) [inline lib/stat_tracker.rb]: This should not have been deleted. It needs to be kept inside worst_offense

File: Gemfile.lock [hunk 3]
```diff
@@ -9,6 +9,7 @@ GEM
     simplecov_json_formatter (0.1.4)
 
 PLATFORMS
+  ruby
```
Comment (DarbySmith) [inline Gemfile.lock]: Do you know why this is here? Mine does not have 'ruby'?

**Review by DarbySmith — COMMENTED** [review]
I saw in the Gemfile.lock a change that added 'ruby' to the file. I am not sure if that was intentional. I do not have that in my Gemfile.lock

**Review by knestler — APPROVED** [review]
good changes!

**Review by DarbySmith — APPROVED** [review]
looks good!

**Pull request merged**